### PR TITLE
DEVDOCS-4367: docs(checkout) CHECKOUT-6874: added original price for response item object

### PR DESCRIPTION
**Pull request by bc-shawnwang**
## What changed?
- Add `originalPrice` for physical/digital item object in storefront cart/checkout api doc response schema and example
- Add `original_price` for physical/digital item object in storefront cart/checkout api doc response schema and example
- Change the description of listPrice/list_price to make it consistent. 

@bigcommerce/checkout

### CODE REVIEW

**Review by bc-mackxu — APPROVED** [review]

**Review by animesh1987 — APPROVED** [review]

**Comment by snaderiBC** [comment]
Also pinging @bigcommerce/dev-docs for visibility

**Comment by snaderiBC** [comment]
@bc-traciporter

**Comment by snaderiBC** [comment]
Thanks @bc-shawnwang 
Please wait for @bc-traciporter or @bigcommerce/dev-docs team approval before merging.

File: reference/carts.sf.yml
Comment (bc-traciporter) [inline reference/carts.sf.yml:660]:
```suggestion
          description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.sf.yml
Comment (bc-traciporter) [inline reference/carts.sf.yml:825]:
```suggestion
          description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.v3.yml
Comment (bc-traciporter) [inline reference/carts.v3.yml:1526]:
```suggestion
                    description: 'The net item price before discounts and coupons. BigCommerce derives an item's list price from the product default price or, if applicable, the sale price configured in the admin panel.'
```

File: reference/carts.v3.yml
Comment (bc-traciporter) [inline reference/carts.v3.yml:1703]:
```suggestion
                    description: 'The net item price before discounts and coupons. BigCommerce derives an item's list price from the product default price or, if applicable, the sale price configured in the admin panel.'
```

File: reference/carts.v3.yml
Comment (bc-traciporter) [inline reference/carts.v3.yml:2179]:
```suggestion
              description: 'The net item price before discounts and coupons. BigCommerce derives an item's list price from the product default price or, if applicable, the sale price configured in the admin panel.'
```

File: reference/carts.v3.yml
Comment (bc-traciporter) [inline reference/carts.v3.yml:2176]:
```suggestion
              description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.v3.yml
Comment (bc-traciporter) [inline reference/carts.v3.yml:2351]:
```suggestion
          description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:1076]:
```suggestion
                      description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:950]:
```suggestion
                      description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:3951]:
```suggestion
                    description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:3836]:
```suggestion
                    description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:3004]:
```suggestion
                            description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:2892]:
```suggestion
                            description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.v3.yml
Comment (bc-traciporter) [inline reference/carts.v3.yml:1999]:
```suggestion
              description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

**Review by bc-traciporter — COMMENTED** [review]
I made some editorial updates to remove passive voice.

File: reference/carts.sf.yml
Comment (bc-shawnwang) [reply reference/carts.sf.yml:825]: Thanks for the feedback, but for this original price, it's actually prodduct's default price which already set in admin panel, and here it's just return that value in response. if begin with `Set ...`, i think they might misunderstand that it tell them to set the price, but they don't need.

**Comment by bc-shawnwang** [comment]
@bc-traciporter changed the description of list price and original price

**Review by bc-traciporter — APPROVED** [review]

**Pull request merged**
